service.yaml

```diff
@@ -1 +1,6 @@
 name: argocd-bot
+tests:
```

Comment:
I'm not sure if the tests section is really necessary, since the actual tests run as part of the build, but having this step does let us confirm that the docker image can run.